src/dialogs/PreviewUploadOverlay.cpp

```diff
@@ -203,3 +203,13 @@ PreviewUploadOverlay::setPreview(const QString &path)
         setLabels(split[1], mime.name(), data_.size());
         init();
 }
+
+void
+PreviewUploadOverlay::keyPressEvent(QKeyEvent *event)
+{
+        if( event->key() == Qt::Key_Escape )
+        {
+            emit aborted();
+            close();
+        }
```

Comment:
I think you should call the parent keyPressEvent handler in all cases, where you don't use the key, so:
```suggestion
        } else {
                QWidget::keyPressEvent(event);
        }
```

Not sure if that matters though, but it may break the enter to send?

Reply:
I used that for a long time and enter to send is not broken at all.